airflow/providers/common/sql/operators/sql.py

```diff
@@ -229,46 +235,65 @@ def __init__(
         **kwargs,
     ):
         super().__init__(conn_id=conn_id, database=database, **kwargs)
-        for checks in column_mapping.values():
-            for check, check_values in checks.items():
-                self._column_mapping_validation(check, check_values)
 
         self.table = table
         self.column_mapping = column_mapping
+        checks_sql = ""
+        for column, checks in self.column_mapping.items():
+            for check, check_values in checks.items():
+                self._column_mapping_validation(check, check_values)
+            checks_list = [*checks]
```

Comment:
This doesn’t seem to be needed? As far as I can tell you can simply do `for check in checks` beflow.